### PR TITLE
Remove csi-driver-lvm

### PR DESCRIPTION
**Pull request by kanikanema**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 
Remove package csi-driver-lvm

###### Change Log  
Baremetal Platform team (Nexus) was the only user of this package. They no longer need it, hence removed.
I checked SPECS and SPECS-EXTENDED, no dependencies on this pacakge.

###### Does this affect the toolchain?  
**NO**

###### Test Methodology
Full pipeline build in progress to check for regressions.

### CODE REVIEW

File: LICENSES-AND-NOTICES/SPECS/data/licenses.json [hunk 1]
```diff
@@ -2188,7 +2188,6 @@
                 "cmake-fedora",
                 "containerd",
                 "coredns",
-                "csi-driver-lvm",
```
Comment (bfjelds) [inline LICENSES-AND-NOTICES/SPECS/data/licenses.json:2191]: does cgmanifest.json need changing?

Comment (bfjelds) [reply LICENSES-AND-NOTICES/SPECS/data/licenses.json:2191]: (for both src and vendor tarballs?)

Comment (kanikanema) [reply LICENSES-AND-NOTICES/SPECS/data/licenses.json:2191]: Yes need to remove the csi entry in cgmanifest.

**Review by mfrw — COMMENTED** [review]
I think we need a rebase.

**Pull request merged**